app/gui/src/project-view/components/GraphEditor/widgets/WidgetFunction/widgetFunctionCallInfo.ts

```diff
@@ -115,7 +115,7 @@ export function useWidgetFunctionCallInfo(
     if (expressionId != null) {
       return {
         expressionId,
-        visualizationModule: WIDGETS_ENSO_MODULE,
+        visualizationModule: "local.NewProject1.Main",
```

Comment:
- This hard coded name [fixes the problem on my sample NewProject1](https://github.com/enso-org/enso/issues/11021#issuecomment-2994045859) project
- of course, this is not the final fix
- IDE guys, how do I get **fully qualified name of currently opened module** here?

Reply:
I can confirm there is 

> serialized for engine using projectNames.serializeProjectPathForBackend.

and one can use it as 795dee4

> store has `moduleProjectPath` field

I cannot find it! Neither `project` nor `projectNames` [defined here](https://github.com/enso-org/enso/blob/158c3f85ab510a6877ff9c6a6c3f6d616b4e87b4/app/gui/src/project-view/components/GraphEditor/widgets/WidgetFunction/widgetFunctionCallInfo.ts#L45) have `moduleProjectPath` field:

![no moduleProjectPath](https://github.com/user-attachments/assets/f55182c9-8c4a-4513-a90e-12a549703784)